### PR TITLE
UX Developer Onboarding

### PR DESCRIPTION
**Pull request by Ctannerweb**

### CODE REVIEW

File: uxd/ux-developer-ramp-up.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  [DockYard ReefPoints Repository]
+  (https://github.com/dockyard/reefpoints)
+* Create a new branch and discover how to add yourself as an author in
+  just one Pull Request with only one commit
+
+## Additional Research
+
+If you don't know the answer to a problem try to solve it yourself
+with Google first. If you are still stumped just ask for some help and anyone
+will be happy to help.
```
Comment (Ctannerweb) [inline uxd/ux-developer-ramp-up.md]: @acacheung Do you want this Ramp Up to be more geared toward DockYard or a more general guide for anyone to use???

**Comment by Ctannerweb** [comment]
@acacheung Do you want this Ramp Up to be geared toward Dockyard or a more broad Ramp Up for anyone to use??

**Comment by acacheung** [comment]
Geared towards DockYard :)

File: uxd/ux-developer-ramp-up.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+* What do you want me to learn next?
+* I don't fully understand this problem even after research, can we pair on it?
+
+## Vim / tmux
+
+Utilize google first to find out what you don't know about tmux and Vim.
+* Complete
+  [Vim Adventures]
+  (http://vim-adventures.com/)
+* Read tmux book that is provided
```
Comment (acacheung) [inline uxd/ux-developer-ramp-up.md]: Include title and author

File: uxd/ux-developer-ramp-up.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+## BEM / CSS
+
+* Read the DockYard
+  [BEM]
```
Comment (acacheung) [inline uxd/ux-developer-ramp-up.md]: "Class Naming Conventions" instead of BEM

File: uxd/ux-developer-ramp-up.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  (https://github.com/dockyard/reefpoints)
+* Create a new branch and discover how to add yourself as an author in
+  just one Pull Request with only one commit
+
+## Additional Research
+
+If you don't know the answer to a problem try to solve it yourself
+with Google first. If you are still stumped just ask for some help and anyone
+will be happy to help.
+* Look over Dockyard Ember projects like
```
Comment (acacheung) [inline uxd/ux-developer-ramp-up.md]: DockYard

**Comment by acacheung** [comment]
also
Review a Sinatra app (link to https://github.com/WickedGoodConferences/ember)
Review a Middleman app (link to https://github.com/dockyard/reefpoints)
Review a Rails app (link to https://github.com/dockyard/website)

File: uxd/ux-developer-ramp-up.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+  [Power Careers]
+  (https://github.com/dockyard/powercareers)
+  and get a feel for how the projects are organized.
+
+* Read any books you can get your hands on like books from
+  [A Book Apart Library]
+  (http://abookapart.com/).
+
```
Comment (acacheung) [inline uxd/ux-developer-ramp-up.md]: no extra line at end of file

File: uxd/ux-developer-ramp-up.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+  __tmux Productive Mouse-Free Development__ - Author, Brian P. Hogan
+
+## BEM / CSS
+
+* Read the DockYard
+  [BEM]
+  (https://github.com/dockyard/styleguides/blob/master/uxd/class-naming-conventions.md#bem-naming-conventions)
+  styleguide
+* Read provided SMACSS epub
```
Comment (acacheung) [inline uxd/ux-developer-ramp-up.md]: link to https://smacss.com/ and author: Jonathan Snook

File: uxd/ux-developer-ramp-up.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+* Complete
+  [Vim Adventures]
+  (http://vim-adventures.com/)
+* Read tmux book that is provided:
+
+  __tmux Productive Mouse-Free Development__ - Author, Brian P. Hogan
+
+## BEM / CSS
+
+* Read the DockYard
```
Comment (acacheung) [inline uxd/ux-developer-ramp-up.md]: Read all UXD styleguides and links included in those styleguides

**Comment by Ctannerweb** [comment]
@acacheung Changes are made and I made some formatting changes

**Comment by acacheung** [comment]
Remove .DS_Store files

File: uxd/ux-developer-ramp-up.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+
+If you don't know the answer to a problem try to solve it yourself
+with Google first. If you are still stumped just ask for some help and anyone
+will be happy to help.
+* Review a DockYard Ember app -
+  [Power Careers]
+  (https://github.com/dockyard/powercareers)
+
+* Review a Sinatra app -
+  [Wicked Good Confrences]
```
Comment (acacheung) [inline uxd/ux-developer-ramp-up.md]: Wicked Good Ember

File: uxd/ux-developer-ramp-up.md [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+  [ReefPoints]
+  (https://github.com/dockyard/reefpoints)
+
+* Review a Rails app -
+  [DockYard Website]
+  (https://github.com/dockyard/website)
+
+* Read any books you can get your hands on like books from
+  [A Book Apart Library]
+  (http://abookapart.com/)
```
Comment (acacheung) [inline uxd/ux-developer-ramp-up.md]: We can remove this since its not required

**Comment by acacheung** [comment]
Looking good! This is almost ready :+1:

**Comment by Ctannerweb** [comment]
Changes are made. Let me know if you see anything else :)

**Pull request merged**